Makefile

```diff
@@ -176,7 +176,10 @@ run-apk-tests:
 	exit $$_r
 
 run-performance-tests:
-	$(call MSBUILD_BINLOG,run-performance-tests,,Test) $(TEST_TARGETS) /t:RunPerformanceTests
+	_r=0 ; \
+	$(call MSBUILD_BINLOG,run-apk-tests,,Test) $(TEST_TARGETS) /t:RunApkTests /p:RunApkTestsTarget=RunPerformanceApkTests $(APK_TESTS_PROP) || _r=$$? ; \
+	$(call MSBUILD_BINLOG,run-performance-tests,,Test) $(TEST_TARGETS) /t:RunPerformanceTests || _r = $$? ; \
```

Comment:
Setting aside the [previous comment](https://github.com/xamarin/xamarin-android/pull/3534/files#r317299053), the spaces around assignment here means that `_r` is *not changed*, and the *overall* pipeline has an exit value of 128:

```bash
_r=0; (exit 1) || _r = 1;
-bash: _r: command not found
$ echo $?
127
```

I'm not entirely sure what that'll do within a Makefile.  The fact that the [Jenkins PR build is green](https://jenkins.mono-project.com/job/xamarin-android-pr-pipeline-release/2043/) is not encouraging...though neither of these `$(call MSBUILD_BINLOG)` statements failed, which means `_r = $$?` wasn't even *evaluated*...